examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -415,6 +415,33 @@ class SubscribeEvent : public SubscribeCommand
     std::vector<chip::EventId> mEventIds;
 };
 
+class ReadNone : public ReadCommand
+{
+public:
+    ReadNone(CredentialIssuerCommands * credsIssuerConfig) : ReadCommand("read-none", credsIssuerConfig)
+    {
+        AddArgument("fabric-filtered", 0, 1, &mFabricFiltered,
+                    "Boolean indicating whether to do a fabric-filtered read. Defaults to true.");
+        AddArgument("data-versions", 0, UINT32_MAX, &mDataVersions,
```

Comment:
It's a bit weird to have data-versions on here, but I guess we could want to send a thing with no paths but included data-versions....